requirements.txt

```diff
@@ -16,4 +16,12 @@ rospkg==1.5.0
 tensorflow
 torch==1.13.1
 tqdm==4.64.0
-transformers[sentencepiece]==4.26.0
\ No newline at end of file
+transformers[sentencepiece]==4.26.0
+catkin_pkg==0.5.2
```

Comment:
could you please sort them as we had in earlier commits? :)
furthermore:
empy and rospkg are already in the requirements, so there is no need to add them

Reply:
done